__tests__/app.test.js

```diff
@@ -137,3 +137,70 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status:200, responds with an array of comments related to the article_id param", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(2);
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```

Comment:
You would need to also check that these comments have the article_id that you've requested

Reply:
Should I add article_id into the expect.objectContaining block? 

I wasn't sure if the returned comment objects have to exclusively include the properties stated on the trello card